events/apigw.go

```diff
@@ -27,14 +27,15 @@ type APIGatewayProxyResponse struct {
 // APIGatewayProxyRequestContext contains the information to identify the AWS account and resources invoking the
 // Lambda function. It also includes Cognito identity information for the caller.
 type APIGatewayProxyRequestContext struct {
-	AccountID    string                    `json:"accountId"`
-	ResourceID   string                    `json:"resourceId"`
-	Stage        string                    `json:"stage"`
-	RequestID    string                    `json:"requestId"`
-	Identity     APIGatewayRequestIdentity `json:"identity"`
-	ResourcePath string                    `json:"resourcePath"`
-	HTTPMethod   string                    `json:"httpMethod"`
-	APIID        string                    `json:"apiId"` // The API Gateway rest API Id
+	AccountID    string                            `json:"accountId"`
+	ResourceID   string                            `json:"resourceId"`
+	Stage        string                            `json:"stage"`
+	RequestID    string                            `json:"requestId"`
+	Identity     APIGatewayRequestIdentity         `json:"identity"`
+	ResourcePath string                            `json:"resourcePath"`
+	Authorizer   APIGatewayCustomAuthorizerContext `json:"authorizer"`
```

Comment:
make this field `Authorizer map[string]interface{}` instead of changing `APIGatewayCustomAuthorizerContext`

Reply:
It was my original code... :) I'll change it

Reply:
Hey @bmoffatt, do you want to keep the old struct in the code?

Reply:
Leave it for now. Technically removing it would be a breaking change ;)